File: app/src/main/java/life/plank/juna/zone/data/network/model/Commentary.java
package life.plank.juna.zone.data.network.model;

import lombok.Data;

@Data
public class Commentary {
    private long id;
    private boolean isImportant;
    private long order;
    private boolean isGoal;
    private long minute;
    private long extraMinute;
    private String comment;
}

File: app/src/main/java/life/plank/juna/zone/data/model/football/Commentary.kt
package life.plank.juna.zone.data.model.football

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Commentary(
        var matchId: Long = 0,
        var isImportant: Boolean = false,
        var order: Long = 0,
        var isGoal: Boolean = false,
        var minute: Long = 0,
        var extraMinute: Long = 0,
        var comment: String = ""
) : Parcelable